### DISCUSSION TITLE
T27 report

### DISCUSSION THREAD

**Discussion by vincent-dp** [simulationcraft/simc#6039] [Q&A]
Hello,

I am confused about the tier27 folders, every gear in each simc files are from the nathria castle. Anything I can do to help here? Also, when will the first 9.1 t27 report will be generated/online? (same, anything I can do to help? All I got is time)

**Comment by Pewtro** [comment]
T26 is Castle Nathria, T27 is SOD. 
T27 uses SOD gear generally everywhere from what I can see, is there anything specific you're missing?
